service/eventer/github/github.go

```diff
@@ -111,28 +152,24 @@ func New(config Config) (*Eventer, error) {
 
 func (e *Eventer) Boot() {
 	e.bootOnce.Do(func() {
-		// TODO check if TPO exists
-
-		// TODO if TPO does not exist
-		// TODO     go to start
-
-		// TODO if TPO exists
-		// TODO     check if TPO's project list is empty
-		// TODO         if TPO's project list is empty
-		// TODO             read all recent deployment events of configured projects
-		// TODO             update TPO with project information
-
-		// TODO loop over deployment events channel and periodically update TPO
+		for {
+			err := e.bootWithError()
+			if err != nil {
+				e.logger.Log("error", fmt.Sprintf("%#v", microerror.Mask(err)))
+			}
+		}
 	})
 }
 
+// TODO create a Github service in a separate PR to ease testing.
 func (e *Eventer) NewDeploymentEvents() (<-chan eventerspec.DeploymentEvent, error) {
 	e.logger.Log("debug", "starting polling for github deployment events", "interval", e.pollInterval)
 
 	deploymentEventChannel := make(chan eventerspec.DeploymentEvent)
 	ticker := time.NewTicker(e.pollInterval)
 
 	go func() {
+		// TODO this map just grows which means we fill the programm with memory.
```

Comment:
This is currently the case in draughtsman and should be fixed here at some point. Not really critical right now though.